PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+<!--
```

Comment:
I think we can make this much snappier. As is (in most of our repos) it's longer than the default size of the text field, which is super clunky. WDYT about this:

```markdown
 - [ ] CHANGELOG.md has been updated

<!--
  If this change has a user visible change (including
  bug fixes, new features, etc) please describe the change in
  CHANGELOG.md.  Consider also updating the README.
 -->
```

Reply:
💯 💯 💯  ![i love it](https://media.giphy.com/media/zTLzS9ABzAAHm/giphy.gif)